Makefile

```diff
@@ -51,8 +51,8 @@ clean: ## Deletes all generated files
 	@echo "+ $@"
 	rm -rf $(JAVA_DIR)  || true
 	rm -rf $(GO_DIR)  || true
-	rm -R `find node -type d \( -iname "*" ! -iname "node_modules" ! -iname "tests" \) -mindepth 1 -maxdepth 1` || true
+	rm -rf `find $(NODE_DIR) -type d \( -iname "*" ! -iname "node_modules" ! -iname "tests" \) -mindepth 1 -maxdepth 1` 2> /dev/null || true
```

Comment:
get rid of usage printout of `rm` when no files need to be deleted by passing the errout to `/dev/null`